pkg/inject/inject.go

```diff
@@ -934,6 +936,24 @@ func (conf *ResourceConfig) applyAnnotationOverrides(values *l5dcharts.Values) {
 		}
 	}
 
+	if override, ok := annotations[k8s.ProxyOutboundDiscoveryCacheTimeout]; ok {
+		duration, err := time.ParseDuration(override)
+		if err != nil {
+			log.Warnf("unrecognized proxy-outbound-discovery-unused-timeout duration value found on pod annotation: %s", err.Error())
+		} else {
+			values.Proxy.OutboundDiscoveryCacheUnusedTimeout = fmt.Sprintf("%ds", int(duration.Seconds()))
+		}
+	}
+
+	if override, ok := annotations[k8s.ProxyInboundDiscoveryCacheTimeout]; ok {
+		duration, err := time.ParseDuration(override)
+		if err != nil {
+			log.Warnf("unrecognized proxy-inbound-discovery-unused-timeout duration value found on pod annotation: %s", err.Error())
```

Comment:
nitpicky, not a big deal: it might be nicer to use the constant with the name of the annotation in these log messages, so that if we decided to change the annotation name later, the log message would reflect that?

Reply:
Hmm, I'm with you on this one. It seems the style is a bit inconsistent throughout the function too. I was debating on whether leaving it like this is better (for consistency reasons). I think a gradual adoption here is good though so I'm fine to change it.